Common/Data/Collections/FastVec.h

```diff
@@ -7,6 +7,7 @@
 
 #include <cstdlib>
 #include <cstring>
+#include <cstdint>
```

Comment:
I was suffering compilation issues when trying to execute it in "DEBUG"

Reply:
That's alright, I probably screwed that up recently in my header cleanup

Reply:
👍